CRM/Member/Page/Tab.php

```diff
@@ -261,6 +261,17 @@ public function view() {
    * @return null
    */
   public function edit() {
+    // We're trying to edit existing memberships or create a new one so we'll first check that a membership
+    // type is configured and active, if we don't do this we instead show a permissions error and status bounce.
+    $membershipTypes = \Civi\Api4\MembershipType::get(TRUE)
+      ->addWhere('is_active', '=', TRUE)
```

Comment:
```suggestion
      ->addWhere('is_active', '=', TRUE)
```

I'm a little unsure of this - I'm thinking of cases where one might want to edit a deactivated membership type and have no other active membership types configured. Is that "too niche" to handle?